Classes/Event/Metadata.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Neos\Cqrs\Event;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * Metadata
+ */
+class Metadata
+{
+    const VERSION = 'neos.cqrs.event:version';
```

Comment:
Changed this value from `neos.eventstore.event:version`. To be discussed, but IMO the event version should be a [core concept of stored events anyways](https://github.com/neos/Neos.EventStore/issues/1#issuecomment-249171648)